src/main/java/parser/Parser.java

```diff
@@ -0,0 +1,89 @@
+package parser;
+
+import command.AddCommand;
+import command.Command;
+import command.DeleteCommand;
+import command.DoneCommand;
+import command.ExitCommand;
+import command.FindCommand;
+import command.ListCommand;
+
+import exceptions.DukeException;
+
+public class Parser {
+    /**
+     * List of Command Strings.
+     */
+    private static final String COMMAND_EXIT = "bye";
+    private static final String COMMAND_LIST = "list";
+    public static final String COMMAND_TODO = "todo";
+    public static final String COMMAND_DEADLINE = "deadline";
+    public static final String COMMAND_EVENT = "event";
+    public static final String COMMAND_DONE = "done";
+    public static final String COMMAND_DELETE = "delete";
+    public static final String COMMAND_FIND = "find";
+
+    private static final String DELIMITER = " ";
+
+    /**
+     * Error Messages.
+     */
+    private static final String ERROR_UNKNOWN_COMMAND = "Unknown Command. See README for full list of commands.";
+
+    /**
+     * Returns the Command corresponding to the full String command input.
+     * String input examples:
+     * <p>
+     * bye
+     * list
+     * todo read book
+     * deadline return book /by 2021-10-15
+     * deadline return book /by 2021-10-15 18:00
+     * event book fest /at 2021-10-15
+     * event book fest /at 2021-10-15 18:00
+     * done 2
+     *
+     * @param fullCommand Full String command input by user.
+     * @return Corresponding command from the input.
+     */
```

Comment:
I like how the comments are so detailed. 👍

Reply:
Thank you! 😄